packages/opentelemetry/lib/plugin.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-case-declarations */
+
+import BasePlugin from '@appium/base-plugin';
+const { tracerProviderInstance } = require('./tracing/tracerProvider');
+
+export default class OpentelemetryPlugin extends BasePlugin {
+  constructor (pluginName, opts = {}) {
+    super(pluginName, opts);
+  }
+
+  static async updateServer (expressApp/*, httpServer*/) { // eslint-disable-line require-await
+    expressApp.post('/opentelemetry/config', OpentelemetryPlugin.setOpentelemetryConfig);
+    expressApp.get('/opentelemetry/config', OpentelemetryPlugin.getOpentelemetryConfig);
+    expressApp.get('/opentelemetry/status', OpentelemetryPlugin.getStatus);
+  }
+
+  static getOpentelemetryConfig (_req, res) {
+    res.send(JSON.stringify(tracerProviderInstance.getCurrentConfig()));
+  }
+
+  static setOpentelemetryConfig (_req, res) {
+    try {
+      const opentelemetryBlob = _req.body;
+      const exporterBlob = opentelemetryBlob.exporter;
+      tracerProviderInstance.generateSpanProcessorForExporter(exporterBlob);
```

Comment:
I assume only this method call can actually throw an error

Reply:
any bad input from the user will throw an error, for.eg if exporter is not defined, etc.

Reply:
I mean we could reduce the size of try/catch block. This helps to improve readability